libevmasm/Assembly.h

```diff
@@ -146,9 +147,7 @@ class Assembly
 	) const;
 
 	/// Create a JSON representation of the assembly.
-	Json::Value assemblyJSON(
-		std::map<std::string, unsigned> const& _sourceIndices = std::map<std::string, unsigned>()
-	) const;
+	Json::Value assemblyJSON(std::map<std::string, unsigned> const& _sourceIndices = std::map<std::string, unsigned>(), bool _includeSourceList = true) const;
```

Comment:
This line is getting long. I'd keep it the way it was split originally:
```suggestion
	Json::Value assemblyJSON(
		std::map<std::string, unsigned> const& _sourceIndices = std::map<std::string, unsigned>(),
		bool _includeSourceList = true
	) const;
```